src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerPushImage.groovy

```diff
@@ -31,8 +32,15 @@ class DockerPushImage extends AbstractDockerRemoteApiTask implements RegistryCre
      * The image name e.g. "bmuschko/busybox" or just "busybox" if you want to default.
      */
     @Input
+    @Optional
     final Property<String> imageName = project.objects.property(String)
 
+    /**
+     * The image names to push e.g. ["bmuschko/busybox"]
+     */
+    @Input
+    final ListProperty<String> imageNames = project.objects.listProperty(String)
```

Comment:
hmmm ... guess I was thinking we would be keeping things more in line with what we did elsewhere in the PR and make the `tag` parameter plural and create a `tags` parameter and deprecate the former.

Reply:
Yeah that makes sense. I can look into doing it that way. 

I originally did it like this because the plugin was autowiring the imageName based on the results of the DockerBuildImage task